<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myresourse.qrc">
    <normaloff>:/Images/AppIcon.png</normaloff>:/Images/AppIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QComboBox {
    border: 1px solid gray;
    border-radius: 3px;
    padding: 1px 18px 1px 3px;
    min-width: 6em;
}

QPushButton {
    border: 2px solid #8f8f91;
    border-radius: 6px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #f6f7fa, stop: 1 #dadbde);
    min-width: 80px;
}

QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f1f1fa);
}

QGroupBox {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #E0E0E0, stop: 1 #FFFFFF);
    border: 2px solid gray;
    border-radius: 8px;
    margin-top: 1ex; /* leave space at the top for the title */
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; /* position at the top center */
    padding: 0 3px;
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FFOECE, stop: 1 #FFFFFF);
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 0 8px;
    background: white;
    selection-background-color: darkgray;
}

QMenuBar {
    background-color: qlineargradient(x1:0, y1:0, x2:0, y2:1,
                                      stop:0 lightgray, stop:1 darkgray);
}

QMenuBar::item {
    spacing: 3px; /* spacing between menu bar items */
    padding: 1px 4px;
    background: transparent;
    border-radius: 4px;
}

QMenuBar::item:selected { /* when selected using mouse or keyboard */
    background: #a8a8a8;
}

QMenuBar::item:pressed {
    background: #888888;
}

QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid #C2C7CB;
}

QTabWidget::tab-bar {
    left: 5px; /* move to the right by 5px */
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>600</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Search</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>19</y>
       <width>140</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Search criterion:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>All</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Name</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Surname</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Rang</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Shop number</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="value_le">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>120</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="search_btn">
     <property name="geometry">
      <rect>
       <x>435</x>
       <y>30</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
     <property name="icon">
      <iconset resource="myresourse.qrc">
       <normaloff>:/Images/Zerode-Plump-Search.ico</normaloff>:/Images/Zerode-Plump-Search.ico</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>600</width>
      <height>200</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_as_Excel"/>
    <addaction name="actionSave_as_Word"/>
    <addaction name="actionSave_as_PDF"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave_as_Excel">
   <property name="icon">
    <iconset resource="myresourse.qrc">
     <normaloff>:/Images/Excel-icon.png</normaloff>:/Images/Excel-icon.png</iconset>
   </property>
   <property name="text">
    <string>Save as Excel</string>
   </property>
  </action>
  <action name="actionSave_as_PDF">
   <property name="icon">
    <iconset resource="myresourse.qrc">
     <normaloff>:/Images/Treetog-File-Type-Pdf.ico</normaloff>:/Images/Treetog-File-Type-Pdf.ico</iconset>
   </property>
   <property name="text">
    <string>Save as PDF</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="myresourse.qrc">
     <normaloff>:/Images/close-icon.png</normaloff>:/Images/close-icon.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave_as_Word">
   <property name="icon">
    <iconset resource="myresourse.qrc">
     <normaloff>:/Images/Word-icon.ico</normaloff>:/Images/Word-icon.ico</iconset>
   </property>
   <property name="text">
    <string>Save as Word</string>
   </property>
  </action>
  <action name="actionPropertis">
   <property name="text">
    <string>Propertis</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>search_btn</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>value_le</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="myresourse.qrc"/>
 </resources>
 <connections/>
</ui>
